lib/gtl_formatter.rb

```diff
@@ -132,6 +169,17 @@ def self.format_icon_column(view, row, controller, clickable)
      :icon2 => icon2}.compact
   end
 
+  def self.os_info_details(item)
+
+    "svg/os-#{item.os_image_name}.svg"
+  end
+
+#   def self.product_name_image(item)
+#     return nil unless item
+
+#     {:title => item.product_name}.compact
+#   end
```

Comment:
Remove this commented out method if we no longer need it.

Reply:
forgot about it, thanks. will remove it.